modin/pandas/base.py

```diff
@@ -3665,6 +3666,95 @@ def __getitem__(self, key):
         else:
             return self._getitem(key)
 
+    def xs(
+        self,
+        key,
+        axis=0,
+        level=None,
+        drop_level: bool = True,
+    ):  # noqa: PR01, RT01, D200
+        """
+        Return cross-section from the Series/DataFrame.
+        """
+        axis = self._get_axis_number(axis)
+        labels = self.axes[axis]
+
+        if isinstance(key, list):
+            # deprecated in pandas, to be removed in 2.0
+            warnings.warn(
+                "Passing lists as key for xs is deprecated and will be removed in a "
+                + "future version. Pass key as a tuple instead.",
+                FutureWarning,
+            )
+
+        if level is not None:
```

Comment:
why do we need additional branching for this? shouldn't it natively work if we replace hardcoded `level=0` everywhere below with the level given us as a parameter?

Reply:
@dchigarev this is how it is done in pandas I guess, let's not deviate much from their version of the code.